src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java

```diff
@@ -1058,15 +1058,15 @@ public Object createValue(UIDefaults table) {
                              "KP_RIGHT", "selectNextColumn",
                           "shift RIGHT", "selectNextColumnExtendSelection",
                        "shift KP_RIGHT", "selectNextColumnExtendSelection",
-                     "ctrl shift RIGHT", "selectNextColumnExtendSelection",
+                     "ctrl shift RIGHT", "selectLastColumnExtendSelection",
                   "ctrl shift KP_RIGHT", "selectNextColumnExtendSelection",
```

Comment:
should KP_RIGHT also be changed to selectLastColumnExtendSelection? Should keypad shortcuts be identical to the arrow key shortcuts?

Reply:
Actually all KP short cuts are miss aligned with native. So I thought of handling it separately.